napari/utils/events/event.py

```diff
@@ -729,12 +724,6 @@ def __init__(
 
     # mypy fix for dynamic attribute access
     def __getattr__(self, name: str) -> EventEmitter:
```

Comment:
if we're removing the warning here... I think you can just remove the whole `__getattr__` method

Reply:
dropped